### DISCUSSION TITLE
护城河还在但城没了 # 组合月报(202107) · BMPI

### DISCUSSION THREAD

**Discussion by giscus** [bmpi-dev/bmpi.dev#134] [Announcements]
# 护城河还在但城没了 # 组合月报(202107) · 构建我的被动收入

被动收入投资组合2021年07月月报

https://www.bmpi.dev/money/passive-income-protfolio/202107/

**Comment by androo789** [comment]
马哥，如果出现政策原因的大跌，还会再等双均线的死叉卖出吗？还是马上就卖

**Reply by madawei2699** [reply]
我尽可能不预测市场，这句话的意思是说我不知道政策会导致市场暴涨还是暴跌，或者毫无反应，因为市场交易者众多。但某些极端事件，比如战争的确会导致相关市场跌入谷底，但这些事件也很难预测，如果被你知道的时候，你还有机会卖出，说明市场对此的反应很平淡，可能并不会暴跌，因为当你知道的时候，市场的交易者绝大部分都知道了，一般情况的话你是没机会卖出的。所以针对此，我的一个优化是，交易系统发出买入信号后，我会提前设置一个止损线，这个止损线一般会低一些，比如我可以容忍5-10%的亏损，一般中短期均线系统不太可能产生如此大的交易亏损，所以也不会过多干扰，这么做的好处在于可以防止极端事件，一般用条件止损单就可以自动设置这种交易规则。

**Reply by androo789** [reply]
谢谢。容我品一品（真回复啊，有种追星成功的感觉hhh）
